### PR TITLE
A lot of improvements and compatibility with newer versions of Django

### PR DESCRIPTION
**Pull request by mrsarm**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -107,17 +105,77 @@ Installation
...
+where you store only the "label" of the tags, not the ids.
+
+So the mapping is as follow:
+
+.. code:: python
+
+   class Blog(models.Model):
+       tags = ArrayField(models.CharField(max_length=255), blank=True, default=list)
+
+Anf the mapping of the table where you have all the possible tags:
```
Comment (mrtizmoatwork) [inline README.rst]: "Anf" -> "And"

**Review by mrtizmoatwork — CHANGES_REQUESTED** [review]
one minor spelling change, the rest is good to go

**Review by mrtizmoatwork — DISMISSED** [review]
looking good

**Comment by mrsarm** [comment]
@mrtizmoatwork please could you approve again? Because this was a fork of a project that I started to edit outside the `polyswarm` GH account, my commits were not signed with my GPG Key, now I rebased the last commit with my signature.

**Review by mrtizmoatwork — DISMISSED** [review]
ok

**Review by mrtizmoatwork — APPROVED** [review]
ok

**Pull request merged**